targets/XPlatCppSdk/source/code/source/playfab/PlayFabPluginManager.cpp

```diff
@@ -79,6 +79,8 @@ namespace PlayFab
     {
 #ifdef PLAYFAB_PLATFORM_XBOX
         return std::shared_ptr<PlayFabIXHR2HttpPlugin>(new PlayFabIXHR2HttpPlugin());
+#elif defined(PLAYFAB_PLATFORM_WINDOWS)
+        return std::shared_ptr<PlayFabWinHttpPlugin>(new PlayFabWinHttpPlugin());
```

Comment:
>std::shared_ptr<PlayFabWinHttpPlugin>(new PlayFabWinHttpPlugin()); [](start = 15, length = 66)

curious why you're using new instead of make_Shared ?

Reply:
To remain compatible with C++ 11 and Paul pointed out that we have customers that required C++ 11 (not later)

---
In reply to: [244876289](https://github.com/PlayFab/SDKGenerator/pull/368#discussion_r244876289) [](ancestors = 244876289)

Reply:
make_shared is C++11 -> make_Unique is not

---
In reply to: [244876570](https://github.com/PlayFab/SDKGenerator/pull/368#discussion_r244876570) [](ancestors = 244876570,244876289)

Reply:
Ah, good catch! I''l fix that


---
In reply to: [244893864](https://github.com/PlayFab/SDKGenerator/pull/368#discussion_r244893864) [](ancestors = 244893864,244876570,244876289)